UnityProject/Assets/Scripts/Mining/SatchelBehaviour.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SatchelBehaviour : MonoBehaviour, IServerInventoryMove
+{
+
+	public HashSet<NamedSlot> CompatibleSlots = new HashSet<NamedSlot>() {
+		NamedSlot.leftHand,
+		NamedSlot.rightHand,
+		NamedSlot.suitStorage,
+		NamedSlot.belt,
+	};
+
+
+	public PlayerScript Player;
+	public ItemStorage Storage;
```

Comment:
These should be made private since they're only used in this class, and you wouldn't want another class to accidentally change it